bin/www

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
+var debug = require('debug')('node-express-gen:server');
+var http = require('http');
+var express=require('express');
+var morgan=require('morgan');
+/**
+ * Get port from environment and store in Express.
+ */
+
+var hostname='192.168.0.11';
+var port = normalizePort(process.env.PORT || '3000');
+app.set('port', port);
+
+
+/**
+ * Create HTTP server.
+ */
+
+var server =express();
+app.use(morgan('dev'));
+
+// http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+app.listen(port,hostname);
+//server.on('error', onError);
+//server.on('listening', onListening);
+
```

Comment:
could you check if we need the express server to be listening to the requests? That is, can you uncomment these 2 lines?